COLLABORATOR_GUIDE.md

```diff
@@ -24,8 +24,10 @@ understand the project governance model as outlined in
 
 ## Issues and Pull Requests
 
-Courtesy should always be shown to individuals submitting issues and
+Courtesy should **always** be shown to individuals submitting issues and
 pull requests to the Node.js project.
+It is recommanded to be extra welcoming to "first-time-contributors",
```

Comment:
recommanded -> recommended ?

Reply:
You could also drop the first four words:

> Be extra welcoming to "first-time-contributors".

I'd use _especially_ in place of _extra_ and lose the unnecessary quotes and hyphens:

> Be especially welcoming to first time contributors.

Reply:
Ack